packages/material-ui/src/RadioGroup/RadioGroup.js

```diff
@@ -76,7 +76,11 @@ RadioGroup.propTypes /* remove-proptypes */ = {
   /**
    * The default value. Use when the component is not controlled.
    */
-  defaultValue: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+  defaultValue: PropTypes.oneOfType([
```

Comment:
This type is wrong. An array of string and number?

Also, string and number are not the only types accepted. Anything that has a valid `.toString` implementation works.